src/pages/NewChatPage.js

```diff
@@ -260,6 +260,7 @@ class NewChatPage extends Component {
                                     confirmButtonText={this.props.translate('newChatPage.createGroup')}
                                     onConfirmSelection={this.createGroup}
                                     placeholderText={this.props.translate('optionsSelector.nameEmailOrPhoneNumber')}
+                                    safeAreaPaddingBottomStyle={safeAreaPaddingBottomStyle}
```

Comment:
We have `safeAreaPaddingBottomStyle` here already, if it's stale please remove it
https://github.com/Expensify/App/blob/8dffdf8a4ce1685b0b5b8cecbfa3938948e3dfcc/src/pages/NewChatPage.js#L247

Reply:
@Santhosh-Sellavel It's needed. The condition you mentioned is when any contact selected in Create group page.
![select](https://user-images.githubusercontent.com/108292595/218521571-291c9a99-fb2b-4585-8c95-6c55f276ed15.png)


My code is when none selected. If my code is removed, it will look like this after scroll down to bottom:
![no select](https://user-images.githubusercontent.com/108292595/218521628-45c84255-c2df-4c4f-b6bf-e92cb0e520fb.png)

Those 2 are never applied at the same time thanks to this line in BaseOptionsSelector:
```
                contentContainerStyles={shouldShowFooter ? [] : [this.props.safeAreaPaddingBottomStyle]}
```

Reply:
> @Santhosh-Sellavel It's needed.

Why it's needed can you clarify?

> Those 2 are never applied at the same time thanks to this line in BaseOptionsSelector:
That's good

Reply:
>  If my code is removed, it will look like this after scroll down to bottom:
 
 @situchan I'm not asking about your code 
 
 About this  `safeAreaPaddingBottomStyle` in this  line https://github.com/Expensify/App/blob/8dffdf8a4ce1685b0b5b8cecbfa3938948e3dfcc/src/pages/NewChatPage.js#L247

Reply:
This is original code in `main` and needed to prevent `Create group` button overlapped with safe area in case when create group chat & any contact selected. `FixedFooter` isn't responsive to safe area.

Reply:
Okay sounds good!